tools/please_go/install/install.go

```diff
@@ -29,15 +29,13 @@ type PleaseGoInstall struct {
 	outDir       string
 	trimPath     string
 
-	additionalLDFlags string
```

Comment:
Couldn't find any reference where this was used, but it seemed that it meant to be used to store any linker flags passed onto the program. I've assigned the `ldFlags` argument to the `collectedLdFlags` member below. It looks like what it was meant to happen, but need a pair of eyes here.